Packs/PrismaCloudCompute/Integrations/PaloAltoNetworks_PrismaCloudCompute/PaloAltoNetworks_PrismaCloudCompute.py

```diff
@@ -1572,6 +1625,101 @@ def get_impacted_resources(client: PrismaCloudComputeClient, args: dict) -> Comm
         raw_response=raw_response
     )
 
+def get_waas_policies(client: PrismaCloudComputeClient, args: dict) -> CommandResults:
+    """
+    Get the WAAS policies.
+    Implement the command 'prisma-cloud-compute-get-waas-policies'
+
+    Args:
+        client (PrismaCloudComputeClient): prisma-cloud-compute client.
+        args (dict): prisma-cloud-compute-get-waas-policies command arguments
+
+    Returns:
+        CommandResults: command-results object.
+    """
+    policies = client.get_waas_policies()
+
+    rules = policies["rules"][0]["applicationsSpec"][0]
+    formatted_waas_policy = [{
+        "SQLInjection": rules["sqli"]["effect"],
+        "CrossSiteScriptingXSS": rules["xss"]["effect"],
+        "OSCommandInjetion": rules["cmdi"]["effect"],
+        "CodeInjection": rules["codeInjection"]["effect"],
+        "LocalFileInclusion": rules["lfi"]["effect"],
+        "AttackToolsAndVulnScanners": rules["attackTools"]["effect"],
+        "Shellshock": rules["shellshock"]["effect"],
+        "MalformedHTTPRequest": rules["malformedReq"]["effect"],
+        "ATP": rules["networkControls"]["advancedProtectionEffect"],
+        "DetectInformationLeakage": rules["intelGathering"]["infoLeakageEffect"]
```

Comment:
genrally please always use `.get('key')` - if one of those keys are not in the response then we will fail on `KeyError` we want to avoid it, that is true for every part of the integration.